migrator/data/DataImporter.java

```diff
@@ -410,16 +410,26 @@ private void importSkippedRelations() {
             skippedRelations.forEach(msg -> {
                 RelationType relType = transaction.concepts().getRelationType(msg.getLabel());
                 if (relType == null) throw TypeDBException.of(TYPE_NOT_FOUND, msg.getLabel());
-                IDMapper.put(msg.getId(), relType.create().getIID());
+                Relation importedRelation = relType.create();
+                idMapper.put(msg.getId(), importedRelation.getIID());
+
+                int ownershipCount = 0;
+                for (DataProto.Item.OwnedAttribute ownership : msg.getAttributeList()) {
+                    Thing attribute = transaction.concepts().getThing(idMapper.get(ownership.getId()));
+                    assert attribute != null;
+                    importedRelation.setHas(attribute.asAttribute());
+                    ownershipCount++;
+                }
+                status.ownershipCount.addAndGet(ownershipCount);
```

Comment:
we must include attribute ownerships that were skipped, along with the relations that were skipped, or the data import is incomplete!